### DISCUSSION TITLE
pnpm without .npmrc

### DISCUSSION THREAD

**Discussion by UROjQ6r80p** [nitrojs/nitro#2707] [General]
Does Nitro fully work in pnpm without .npmrc `shamefully-host=true`? 
I've seen some related issues in Nuxt about having to manually add `h3` however I just installed Nitro and both `dev` and `build` command worked without having to manually install `h3`
Are there any hidden issues with it along the way? I just want it to work without .npmrc in pnpm monorepo

**Comment by pi0** [comment]
You can directly use nitro without shamefully-hoist with no worries.

**Reply by UROjQ6r80p** [reply]
May I only briefly ask why is `.npmrc` included by default?
Like is it included so that Nuxt can be later added to the mix or something or is it going to be dropped in the future from nitro starter template?
I'm asking cuz after `pnpm dlx giget@latest nitro nitro-app --install` it showed `The modules directory at "C:/.../monorepo/node_modules" will be removed and reinstalled from scratch. Proceed? (Y/n) ` and it's the first time i've seen that in my monorepo, and of course it's a nitpick but it would be nice if nitro project creation went seamlessly without it like with `pnpm create astro@latest` or `pnpm create next-app` etc.
